resources/console/charts/web/values.yaml

```diff
@@ -35,6 +35,7 @@ core_ui:
   resources: {}
 managementPlane: {}
 replicaCount: 1
+resources: {}
```

Comment:
resources are made for console and core-ui separately (lines 25 & 35)

Reply:
something went wrong at merge 🤦